### ISSUE TITLE
Version with lower baudrate

### ISSUE THREAD

**Issue by Niko-O** [kongr45gpen/i2c-sniffer#1]
Could you possibly compile a version with a lower baudrate (no more than 128000)? My USB to serial adapter can't handle much more but I can't compile this code. The I²C-bus I am trying to monitor also runs at a leisurly ~40kHz, so the lower baudrate should not be a problem.

I would greatly appreciate it.

**Comment by kongr45gpen** [comment]
Can you try this binary? It should be running at 128 kbaud/s

[i2c_sniffer.elf.zip](https://github.com/kongr45gpen/i2c-sniffer/files/4219651/i2c_sniffer.elf.zip)

**Comment by Niko-O** [comment]
Yes, thank you! (Though for now, the sniffer is onl receiving garbage even though all devices seem to function normally. I'll investigate.)
